part-2-server/server.js

```diff
@@ -0,0 +1,30 @@
+var http = require('http');
+var fs = require('fs');
+
+
+var server = http.createServer(function(request, response){
+	var url = request.url;
+	if(url === "/"){
+			fs.readFile('index.html', function(err, data){
+				response.write(data.toString())
+				response.end()
+		})
+	} else if (url === '/page-2.html' || url === '/page-3.html'){
+		fs.readFile('page-2.html', function(err, data){
+			response.write(data.toString())
+			response.end()
+		})
+	} 
+
+	for(var i = 0; i < 140; i ++){
+	var poemURL = "/poem-" + i + ".txt"
+	if( url === poemURL){
+		fs.readFile('poem-' + i + ".txt", function(err, data){
+			response.write(data.toString());
+			response.end()
+		})
+	}
+	
```

Comment:
Any way you can make it more efficient? In other words, any way you can do it without having to loop with every request? Hint: you can break the original url into substrings...